boards/arm/stm32u5a9j_dk/stm32u5a9j_dk.dts

```diff
@@ -0,0 +1,318 @@
+	msi-range = <4>; /* 4MHz (reset value) */
+	msi-pll-mode;
+};
+
+&pll1 {
+	div-m = <1>;
+	mul-n = <80>;
+	div-p = <2>;
+	div-q = <2>;
+	div-r = <2>;
+	clocks = <&clk_msis>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll1>;
+	clock-frequency = <DT_FREQ_M(160)>;
+	ahb-prescaler = <1>;
+	apb1-prescaler = <1>;
+	apb2-prescaler = <1>;
+	apb3-prescaler = <1>;
+};
+
+&usart1 {
+	pinctrl-0 = <&usart1_tx_pa9 &usart1_rx_pa10>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+uart0: &usart3 {
+	pinctrl-0 = <&usart3_tx_pb10 &usart3_rx_pb11>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&lpuart1 {
+	pinctrl-0 = <&lpuart1_tx_pg7 &lpuart1_rx_pg8>;
+	pinctrl-names = "default";
+	current-speed = <9600>;
+	status = "okay";
+};
+
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pg14 &i2c1_sda_pg13>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+};
+
+&i2c2 {
+	pinctrl-0 = <&i2c2_scl_pf1 &i2c2_sda_pf0>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+};
+
+&i2c6 {
+	pinctrl-0 = <&i2c6_scl_pd1 &i2c6_sda_pd0>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+};
+
+&spi2 {
+	pinctrl-0 = <&spi2_sck_pb13 &spi2_miso_pd3 &spi2_mosi_pd4>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpiob 12 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+	status = "okay";
+};
+
+&spi3 {
+	pinctrl-0 = <&spi3_sck_pg9 &spi3_miso_pg10 &spi3_mosi_pg11>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpiog 15 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+	status = "okay";
+};
+
+&timers1 {
+	st,prescaler = <1>;
+	status = "okay";
+
+	pwm1: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim1_ch2_pe11>;
+		pinctrl-names = "default";
+	};
+};
+
+&timers2 {
+	st,prescaler = <1>;
+	status = "okay";
+
+	pwm2: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim2_ch4_pa3>;
+		pinctrl-names = "default";
+	};
+};
+
+&sdmmc1 {
+	pinctrl-0 = <&sdmmc1_d0_pc8 &sdmmc1_d1_pc9
+		&sdmmc1_d2_pc10 &sdmmc1_d3_pc11
+		&sdmmc1_d4_pb8 &sdmmc1_d5_pb9
+		&sdmmc1_d6_pc6 &sdmmc1_d7_pc7
+		&sdmmc1_ck_pc12 &sdmmc1_cmd_pd2>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&sdmmc2 {
+	pinctrl-0 = <&sdmmc2_d0_pb14 &sdmmc2_d1_pb15
+		&sdmmc2_d2_pb3 &sdmmc2_d3_pb4
+		&sdmmc2_ck_pd6 &sdmmc2_cmd_pd7>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&adc1 {
+	pinctrl-0 = <&adc1_in5_pa0 &adc1_in14_pc5>;
+	pinctrl-names = "default";
+	st,adc-clock-source = <ASYNC>;
+	st,adc-prescaler = <1>;
+	status = "okay";
+
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	channel@5 {
+		reg = <5>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,resolution = <14>;
+	};
+
+	channel@14 {
+		reg = <14>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,resolution = <14>;
+	};
+};
+
+adc2: &adc1 {
```

Comment:
I'm pretty sure this should simply be `&adc2` here.
But that would need to add the adc2 node in stm32u595.dtsi

Reply:
From section "Analog-to-digital converter (ADC12)" in https://www.st.com/resource/en/reference_manual/rm0456-stm32u5-series-armbased-32bit-mcus-stmicroelectronics.pdf : "STM32U535/545/575/585 devices embed one analog-to-digital converter (ADC1) while STM32U59x/5Ax/5Fx/5Gx devices embed two analog-to-digital converters (ADC1 and
ADC2), controlled by a single interface ADC12. (...) ADC1 and ADC2 are tightly coupled and can operate in dual mode (ADC1 is a master)."

Reply:
They are still two different ADC, they can work together, they can also work separately, and the each have their own registers.
You can take a look at stm32h7.dtsi, we have the same kind of thing for ADC, with adc1, adc2 and adc1_2 (when coupled)

Reply:
Page 146 in https://www.st.com/resource/en/reference_manual/rm0456-stm32u5-series-armbased-32bit-mcus-stmicroelectronics.pdf
![Screenshot from 2023-11-17 10-16-11](https://github.com/zephyrproject-rtos/zephyr/assets/128408655/6a5100a2-eee4-44c1-aead-46327bcef26d)

Reply:
Page 1363
![image](https://github.com/zephyrproject-rtos/zephyr/assets/103205175/dcfce6f5-ecb7-4db7-9474-fe20962c246c)